securing_apps/topics/token-exchange/token-exchange.adoc

```diff
@@ -158,14 +163,32 @@ This setup page displays.
 .Target Client Exchange Permission Setup
 image:{project_images}/exchange-target-client-permission-setup.png[Target Client Exchange Permission Setup]
 
+ifeval::[{project_community}==true]
 . Define a policy for this permission by clicking the *Authorization* link
+endif::[]
+ifeval::[{project_product}==true]
+. Define a policy for this permission by clicking the *Client details* link
```

Comment:
I don't see the Client details link on this screen.

Reply:
The new image exchange-target-client-permission-setup.png is under keycloak-images.